lib/llm/Cargo.toml

```diff
@@ -91,7 +91,7 @@ dialoguer = { version = "0.11", default-features = false, features = ["editor",
 
 # block_manager
 nixl-sys = {version = "0.4.1", optional = true }
-cudarc = { version = "0.16.2", features = ["cuda-12020"], optional = true }
+cudarc = { version = "0.16.6", features = ["cuda-12020"], optional = true }
```

Comment:
FYI @ryanolson for viz since I saw you mention cudarc around cuda streams <-> torch for KVBM